ext/http/00_serve.ts

```diff
@@ -740,7 +740,7 @@ function serve(arg1, arg2) {
   }
 
   const listenOpts = {
-    hostname: options.hostname ?? "0.0.0.0",
+    hostname: options.hostname ?? "localhost",
```

Comment:
This seems to fail a bunch of tests. It might be good to make this a display only thing.